### PR TITLE
Update uad_lists.json

### PR DESCRIPTION
**Pull request by MayankSharma-dev**
Updated UAD lists.

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -36252,5 +36252,53 @@
...
+      "labels": [],
+      "removal": "Recommended"
+    },
+    "com.vanced.android.apps.youtube.music":{
+      "list": "Misc",
+      "description": "Youtube Vanced Music",
+      "dependencies": [],
+      "neededBy": [],
+      "labels": [],
+      "removal": "Recommended"
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:36294]: Uhm, pretty sure these are all user-installed apps. Especially (Re)Vanced and Steam of course. Why did you include those? It's only for bloatware that the user did not install himself.

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: resources/assets/uad_lists.json (hunk 1 above)
Comment (MayankSharma-dev) [reply resources/assets/uad_lists.json:36294]: > Uhm, pretty sure these are all user-installed apps. Especially (Re)Vanced and Steam of course. Why did you include those? It's only for bloatware that the user did not install himself.

true, above-mentioned apps are user-installed but there are times when user just forget or someone else could have installed that on their behalf, not everyone is known to these facts, so it is just for informational purposes and even Opera Browser are preinstalled in some devices. I thought that the uad_list is some kind of community database which contains info about every known apps through which many developers and general users could take the advantage.

Comment (AnonymousWP) [reply resources/assets/uad_lists.json]: That's not the purpose of this project. After all, the apps can just be uninstalled again by the user via the settings of Android. So I won't be adding that. But yeah, this project is not for all apps but only pre-installed apps.

Maybe we should clarify this further.

Comment (MayankSharma-dev) [reply resources/assets/uad_lists.json]: Understood. Thanks for the clarification.

**Pull request closed without merging**